.github/workflows/release_workflow.yml

```
@@ -0,0 +1,108 @@
+      version_file: 'poorman_handshake/version.py'
+      setup_py: 'setup.py'
+      update_changelog: true
+      publish_prerelease: true
+      changelog_max_issues: 100
+
+  notify:
+    if: github.event.pull_request.merged == true
+    needs: publish_alpha
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)